.sail/Dockerfile

```
@@ -1,14 +1,17 @@
 FROM codercom/ubuntu-dev-go:latest
 
-RUN sudo apt-get install -y htop
+RUN sudo apt-get update && \
+  sudo apt-get install -y htop && \
+  curl -sL https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | sudo -E bash -
```

We should Install the latest nvm and install the default node/npm as well. We should also install without `sudo` and in a separate `RUN` to utilize docker build caching
```suggestion
RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash && \
    . ~/.nvm/nvm.sh \
    && nvm install node
```

---

didn't even see that there is a newer version, this was just from the link that you sent before.